tests/src/test/scala/packages/deployWebTests.scala

```diff
@@ -73,7 +73,7 @@ class DeployWebTests extends TestHelpers
       wsk.action.get(deployAction, FORBIDDEN)
     }
 
-    //test to create the hello world blueprint from github
+    //test to create the hello world template from github
```

Comment:
Should make the spacing after `//` consistent among this file and the other Scala file.